nilearn/_utils/testing.py

```diff
@@ -585,12 +615,31 @@ def generate_group_sparse_gaussian_graphs(
     return signals, precisions, topology
 
 
-def skip_if_running_nose(msg=None):
+def skip_if(skip, skip_msg=None):
```

Comment:
From what I could see this was only used for the Neurodebian-specific warning. I think we can live without for now.

Quite annoying though, that there is no way to have a unittest.skipIf in python 2.6 ...